platformio.ini

```diff
@@ -176,9 +176,24 @@ esp32_platform = espressif32@6.1.0
 esp32_solo_platform = https://github.com/tasmota/platform-espressif32/releases/download/v.2.0.5/platform-espressif32-v.2.0.5.zip
 atmelavr_platform = atmelavr@3.3.0
 
-[com-esp]
+[com-esp] ; Used by all ESP8266/ESP8285 based builds
 lib_deps =
   ${env.lib_deps}
+build_flags =
+  ${env.build_flags}
+  '-DENV_NAME="$PIOENV"'
+  '-DZmqttDiscovery="HADiscovery"'
+  '-DTRACE=1'
+  '-DCONFIG_BT_NIMBLE_ROLE_PERIPHERAL_DISABLED'
+  '-DCONFIG_BT_NIMBLE_ROLE_BROADCASTER_DISABLED'
```

Comment:
You can remove this for ESP8266 as BT is not supported on this board

Reply:
Done